armi/reactor/converters/geometryConverters.py

```diff
@@ -1346,6 +1367,11 @@ def restorePreviousGeometry(self, r=None):
         .. impl:: Restore a one-third-core geometry to a full-core geometry.
             :id: I_ARMI_THIRD_TO_FULL_CORE1
             :implements: R_ARMI_THIRD_TO_FULL_CORE
+
+            This method is a reverse process of the method ``convert``. It converts
+            the full-core model back to the original one-third core model by removing
```

Comment:
```suggestion
            the full-core reactor model back to the original one-third core reactor model by removing
```
Along the lines of tony's edits above.